docs/internal/Element-Type-Specs.md

```diff
@@ -331,17 +331,18 @@ To handle objects returned that don't have the required fields, you can use the
 - **type**: `string` -- Can be "toggle" to display as a toggle switch, or "slider" to display as a slider switch (defaults to regular checkbox).
 - **layout**: `string` -- if "inline", displays checkboxes horizontally in rows. Useful if there are a lot of checkboxes.
 - **resetButton**: `boolean` -- if `true`, element will show a "Reset" button, which allows user to reset selections to the initial (loading) state. (Default: `false`)
-- **textDisplay**: (Options: `text`, `list` (default), `propertyList`) -- specifies how to show the applicant's response on the Summary page:
+- **displayFormat**: (Options: `text`, `list` (default), `checkboxes`, `propertyList`) -- specifies how to show the applicant's response on the Summary page:
   - `list`: shows the selected checkboxes in a (Markdown) list
   - `text`: shows the selected values in a comma-seperated text string
+  - `checkboxes`: displays the selected values as checkboxes as per the application view
   - `propertyList`: displays in a list with properties (the checkbox `label` field) and values (the checkbox `text` or `textNegative` values, depending on selection).  
-  e.g.  
-    – Option1: YES
+  e.g.
+    - Option1: YES
     - Option 2: NO
     - \<checkbox `label`\>: \<`text`\/`textNegative` value\>  
-Note: this display option is only suitable if you have separately defined `label`, `text` and `textNegative` fields for each checkbox. 
-- **keyMap**: `object` -- if the input `checkboxes` property (above) has different property names that what is required (for example, if pulling from an API), then this `keyMap` parameter can be used to re-map the input property names to the requried property names. For example, if your input "checkbox" data contained an array of objects of the type `{ name: "Nicole", active: true}`, you would provide a `keyMap` object like this:
+Note: this display option is only suitable if you have separately defined `label`, `text` and `textNegative` fields for each checkbox.
```

Comment:
Are these properties to be set in each element on object `checkboxes`? The phrase is a bit hard to understand and also what is ` - \<checkbox `label`\>: \<`text`\/`textNegative` value\> ` Make add an example?

Reply:
It's clearer if you see it in Preview:
<img width="680" alt="Screen Shot 2022-08-25 at 7 20 32 PM" src="https://user-images.githubusercontent.com/5456533/186600963-8f7bc9fd-c206-4fb9-b167-8a5e76905f80.png">